src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -866,6 +867,16 @@ emit_sys_numerics_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSigna
 	case SN_ConvertToInt64:
 	case SN_ConvertToUInt32:
 	case SN_ConvertToUInt64:
+		break;
+	case SN_Equals:
+		MonoType *type;
```

Comment:
Needs a {} block

Reply:
I moved the declaration out of the switch statement in the second commit. Do you think it is better to keep the declaration inside the switch statement?